### PR TITLE
Feature/new markers

### PR DESCRIPTION
**Pull request by Olga-Mishareva**

### CODE REVIEW

File: components/ConfirmPopup.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import styled, { css } from "styled-components";
+
+export default function ConfirmPopup({ onConfirm, onClose }) {
+  return (
+    <Wrapper onClick={(e) => e.stopPropagation()}>
```
Comment (mbosselmann) [inline components/ConfirmPopup.js:5]: Instead of writing `e` I would prefer to name event `event`. :)

File: components/ConfirmPopup.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+      <ConfirmQuestion>Do you want to notice new item? </ConfirmQuestion>
+      <ButtonsWrapper>
+        <Button variant="confirm" onClick={onConfirm}>
```
Comment (mbosselmann) [inline components/ConfirmPopup.js:8]: Please add a type for every button. Maybe if you add type="button" you do not need `event.stopPropagation()` anymore as the default type of each button is submit. 🤔

File: components/Map/Click.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+const { useMapEvents } = require("react-leaflet");
+
+export default function Click({ onPosition, setIsConfirmPopupOpen }) {
+  const map = useMapEvents({
+    click: (e) => {
```
Comment (mbosselmann) [inline components/Map/Click.js:5]: I would also prefer to write `event` instead of `e` for better readability.

File: components/Map/Map.js [hunk 4]
```diff
@@ -3,18 +3,47 @@ import "leaflet/dist/leaflet.css";
...
+        onPosition={onPosition}
+        setIsConfirmPopupOpen={setIsConfirmPopupOpen}
+      />
+
+      {isConfirmPopupOpen && (
+        <Popup position={clickPosition} closeButton={false}>
+          <ConfirmPopup onConfirm={handleConfirm} onClose={handleClose} />
+        </Popup>
+      )}
+
```
Comment (mbosselmann) [inline components/Map/Map.js:46]: We think this solution is a little bit complicated and you could handle the Popup like in the following example: https://github.com/doemser/nextjs-react-leaflet-map/blob/main/components/Map.jsx#L36-L54
The logic of the popup should be tied to the click and not split into two components that cannot function independently from each other.

File: components/Map/Map.js [hunk 5]
```diff
@@ -3,18 +3,47 @@ import "leaflet/dist/leaflet.css";
...
+  function handleConfirm() {
+    setIsConfirmPopupOpen(false);
+    router.push("/create");
+  }
+
+  function handleClose() {
+    setIsConfirmPopupOpen(false);
+  }
+
+  console.log(isConfirmPopupOpen);
```
Comment (mbosselmann) [inline components/Map/Map.js:29]: The conosle.log should be removed 👀

**Review by mbosselmann — COMMENTED** [review]
Hi Olga,
you rock! 🪨 I am really impressed about your progress. :octocat: 
I left some remarks for you to consider before moving further. :)

File: components/Map/Map.js (hunk 4 above)
Comment (Olga-Mishareva) [reply components/Map/Map.js:46]: Hi Mareike, 
thank you for reviewing and your tips!

**Review by niklasrammelkamp — APPROVED** [review]
nice one

**Pull request merged**